src/css/profile/mobile/common/slider.less

```diff
@@ -82,21 +62,18 @@
 	}
 
 	&.ui-disabled {
+		opacity: 1;
 		.ui-slider-bar {
-			border-image: url(images/3_Controllers/tw_progress_bar_d.9.png) 3 fill;
-			border-image-width: 1 * @px_base;
-			opacity: 0.15;
```

Comment:
Why do you remove the opacity?

Reply:
"background-color: var(--slider-bg-disabled-color)" was defined by  "fade(@slider-bg-disabled-color, @slider-bg-disabled-opacity);
So the "slider-bg-disabled-color" is already have the opacity 0.15 value. (slider-bg-disabled-opacity is 15%)
So I removed.